tools/src/main/java/org/apache/rocketmq/tools/command/stats/StatsAllSubCommand.java

```diff
@@ -94,22 +94,22 @@ public static void printTopicDetail(final DefaultMQAdminExt admin, final String
                 if (!activeTopic || (inMsgCntToday > 0) ||
                     (outMsgCntToday > 0)) {
 
-                    System.out.printf("%-32s  %-32s %12d %11.2f %11.2f %14d %14d%n",
-                        UtilAll.frontStringAtLeast(topic, 32),
-                        UtilAll.frontStringAtLeast(group, 32),
-                        accumulate,
-                        inTPS,
-                        outTPS,
-                        inMsgCntToday,
-                        outMsgCntToday
+                    System.out.printf("%-64s  %-64s %12d %11.2f %11.2f %14d %14d%n",
+                            UtilAll.frontStringAtLeast(topic, 64),
```

Comment:
the indent of code does not match `rmq_codeStyle.xml`
see https://rocketmq.apache.org/docs/code-guidelines/

Reply:
fix code indentation